lib/gitsh/environment.rb

```diff
@@ -88,7 +87,7 @@ def repo_has_untracked_files?
     end
 
     def repo_config_color(name, default)
-      if color_override = self[name]
+      if color_override = fetch(name) { nil }
```

Comment:
oh wait, I think I may have misunderstood. This is if the KEY doesn't exist, not if it doesn't have a value. Nevermind then! :)